src/typeDefs/queries.ts

```diff
@@ -119,5 +119,7 @@ export const queries = gql`
       skip: Int
       orderBy: UserOrderByInput
     ): [User]! @auth
+
+    venuesInOrganization(id: ID!): [Venue] @auth
```

Comment:
No need for this. Add `venues` as a field on the organization model, and then use the field level resolver for the same.